Add CC for LLNL SWING on MSLs

markcmiller86 opened the pull request:
CC article for LLNL SWING presentation (video and slides) regarding memory safe languages.

# Description

EB Member: @rinkug 

Resolves #1984 

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [x] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [x] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [x] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [x] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [x] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

<!-- Standard links below, leave these this section! -->

[preview]: https://preview.bssw.io
[bssw.io]: https://bssw.io
[Content Development]: https://github.com/betterscientificsoftware/bssw.io/projects/3
[BSSw Internal]: https://github.com/betterscientificsoftware/bssw.io/projects/2
[meta-data]: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_common.html#metadata-section
[wikize_refs.py]: https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy

markcmiller86 commented:
This looks ready to me

markcmiller86 commented:
I added `external contribution` label because its based almost entirely on an email exchange the @lindstro

pagrubel commented:
> I added `external contribution` label because its based almost entirely on an email exchange the @lindstro

Should he be added as an author?

markcmiller86 commented:
> > I added `external contribution` label because its based almost entirely on an email exchange the @lindstro
> 
> Should he be added as an author?

Sorry...added comment and label to wrong ticket.

markcmiller86 commented:
@betterscientificsoftware/bssw-editorial-board this is ready to publish, please review and if approved merge.

rinkug reviewed (approved):
I made a few changes.

markcmiller86 commented:
Hmmm...thanks for doing this @rinkug. There is something funky with the slides link now...oh, wait, since its merged to main now, that link needs to be changed. Lemme go fix that.
